### ISSUE TITLE
Got warnings after updating to 1.21.0: Class path contains multiple SLF4J bindings

### ISSUE THREAD

**Issue by fikr4n** [detekt/detekt#5265]
## Expected Behavior

There should be no warning.

## Observed Behavior

```
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/user/.gradle/caches/modules-2/files-2.1/io.gitlab.arturbosch.detekt/detekt-formatting/1.21.0/b7e14d2040b8c6092227a0056223e9f36cc54a6/detekt-formatting-1.21.0.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/user/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-nop/1.7.36/a3c1eb685d59414527faa93623acae311c184032/slf4j-nop-1.7.36.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.helpers.NOPLoggerFactory]
```

## Steps to Reproduce

Run `./gradlew detekt` where the config is like below:

```groovy
task detekt(type: JavaExec, group: "verification") {
    mainClass.set("io.gitlab.arturbosch.detekt.cli.Main")
    classpath = configurations.detekt
    def params = ['-i', ..., '-c', ..., '-r', ..., '--build-upon-default-config']
    args(params)
}
```

## Context

Simply running detekt.

## Your Environment

* Version of detekt used: 1.21.0
* Version of Gradle used (if applicable): 7.2.0
* Operating System and version: Ubuntu 22.04

**Comment by fikr4n** [comment]
Thanks
